sdcm/ycsb_thread.py

```diff
@@ -157,21 +157,26 @@ def copy_template(self, cmd_runner, loader_name, memo={}):  # pylint: disable=da
                     dynamodb.primaryKey = {alternator.consts.HASH_KEY_NAME}
                     dynamodb.primaryKeyType = {alternator.enums.YCSBSchemaTypes.HASH_SCHEMA.value}
                 ''')
-
-            aws_empty_file = dedent(f""""
-                accessKey = {self.params.get('alternator_access_key_id')}
-                secretKey = {alternator.api.Alternator.get_salted_hash(node=self.node_list[0], username=self.params.get('alternator_secret_access_key'))}
-            """)
+            if self.params.get('alternator_enforce_authorization'):
+                aws_credentials_file = dedent(f""""
```

Comment:
`aws_credentials_file` name is confusing as it refers to content, while next to it we use `aws_dummy_credentials_file` which refers to file path.

Reply:
fixed